packages/webpack-config/webpack/webpack.prod.js

```diff
@@ -57,12 +57,14 @@ module.exports = async function(env = {}, argv) {
               // https://github.com/terser-js/terser/issues/120
               inline: 2,
             },
-            mangle: {
-              safari10: true,
-            },
+            mangle: process.env.EXPO_WEB_DEBUG
+              ? false
+              : {
+                  safari10: true,
+                },
             output: {
               ecma: 5,
-              comments: false,
+              comments: process.env.EXPO_WEB_DEBUG,
```

Comment:
```suggestion
              comments: getenv.boolish('EXPO_WEB_DEBUG', false),
```
(Environment variables are always strings so they need to be converted to boolean.)